scss/colors.scss

```diff
@@ -0,0 +1,5 @@
+$general-background-color: #F4F4F4;
```

Comment:
> this color variables should describe how the color looks and not where it is used


Check the tech-guides, and [Name that color](http://chir.ag/projects/name-that-color/) page